### PR TITLE
docs: Add more examples for multiline-ternary

### PR DESCRIPTION
**Pull request by AshiotisCy**
I have thought of a new way to implement the multiline-ternary.

I have moved the ? and : below the logical expression to make it more readable and understandable in cases were logical expressions get more complex.

Also this way has a look of a tree to know where the first "IF" statement starts and where it finishes.

#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

**Comment by mdjermanovic** [comment]
Hi @AshiotisCy, thanks for the PR!

> I have thought of a new way to implement the multiline-ternary.

Can you please clarify the intent of this PR, is it to change the way the rule works or just add more examples of how the rule already works?

**Comment by AshiotisCy** [comment]
> Hi @AshiotisCy, thanks for the PR!
> 
> > I have thought of a new way to implement the multiline-ternary.
> 
> Can you please clarify the intent of this PR, is it to change the way the rule works or just add more examples of how the rule already works?

Its basically to change the view the multiline-tenary works for 

> Hi @AshiotisCy, thanks for the PR!
> 
> > I have thought of a new way to implement the multiline-ternary.
> 
> Can you please clarify the intent of this PR, is it to change the way the rule works or just add more examples of how the rule already works?

Is to change the way the rule works. I find the way i provided easier to read and understand since it will provide a more clear way when you have multiple line with ternary.

The example i provided is to give a more clear vision 
`foo > bar
    ? (baz > qux
        ? value1
        : value2)
    : value3;`
 
 rather than:
 
 `foo > bar ?
    (baz > qux ?
        value1 :
        value2) :
    value3;`
    
   With the 1 option you can see where the first '?' starts along with its closing ':'

**Comment by mdjermanovic** [comment]
Stylistic rules are [frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we're not making any changes to the way the `multiline-ternary` rule works.

If you'd like, we can accept adding more correct/incorrect examples of how the rule already works, but I think we should keep the existing examples as well.

**Comment by mdjermanovic** [comment]
> ```
> foo > bar
>     ? (baz > qux
>         ? value1
>         : value2)
>     : value3;
> ```
> 
> 
>     
>   
> 
> rather than:
> 
> 
> ```
>  foo > bar ?
>     (baz > qux ?
>         value1 :
>         value2) :
>     value3;
> ```

Note that this is controlled by the [operator-linebreak](https://eslint.org/docs/latest/rules/operator-linebreak) rule. As for the `multiline-ternary` rule, both examples are valid.

**Review by mdjermanovic — CHANGES_REQUESTED** [review]

**Comment by AshiotisCy** [comment]
> Stylistic rules are [frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we're not making any changes to the way the `multiline-ternary` rule works.
> 
> If you'd like, we can accept adding more correct/incorrect examples of how the rule already works, but I think we should keep the existing examples as well.

Ok then lets just add it as another correct example

File: docs/src/rules/multiline-ternary.md [hunk 1]
```diff
@@ -69,11 +71,11 @@ foo > bar ?
     value1 :
     value2;
 
-foo > bar ?
-    (baz > qux ?
-        value1 :
-        value2) :
-    value3;
```
[the remark this answers is not in the document]
Comment (AshiotisCy) [reply docs/src/rules/multiline-ternary.md]: Done

File: docs/src/rules/multiline-ternary.md [hunk 2]
```diff
@@ -49,11 +54,8 @@ Examples of **incorrect** code for this rule with the `"always"` option:
 
 foo > bar ? value1 : value2;
 
-foo > bar ? value :
-    value2;
-
-foo > bar ?
-    value : value2;
```
[the remark this answers is not in the document]
Comment (AshiotisCy) [reply docs/src/rules/multiline-ternary.md]: Done

File: docs/src/rules/multiline-ternary.md [hunk 3]
```diff
@@ -164,9 +172,11 @@ foo > bar ? value1 : value2;
 
 foo > bar ? (baz > qux ? value1 : value2) : value3;
 
-foo > bar ? (
-    baz > qux ? value1 : value2
-) : value3;
```
[the remark this answers is not in the document]
Comment (AshiotisCy) [reply docs/src/rules/multiline-ternary.md]: Done

File: docs/src/rules/multiline-ternary.md [hunk 4]
```diff
@@ -164,9 +172,11 @@ foo > bar ? value1 : value2;
...
+foo > bar
+    ? ( baz > qux
+        ? value1
+        : value2 )
+    : value3;
```
[the remark this answers is not in the document]
Comment (AshiotisCy) [reply docs/src/rules/multiline-ternary.md]: You are right, I have removed it from the the "never" example

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**